### DISCUSSION TITLE
[Projects Beta] add all issues from all user/org's repos

### DISCUSSION THREAD

**Discussion by OJFord** [community/community#6258] [Projects and Issues]
I'd like to be able to easily add all existing issues/PRs to a project, and then continually add them automatically going forward.

Currently, this seems to be harder than it was in the old Projects - the '+' button there surfaces all issues from non-archived repositories, whereas the new Projects (beta) requires choosing a repo name first, so one must go through the alphabet checking every repo for issues that haven't been added.

I realise that it's somewhat possible (other than the initial import) with an Actions job in each repo, but that's a pain to go back through adding them to all existing repos, and then to remember to add to new repos going forward. (And then if there's a bug, update them all separately again?!)

It'd be nice to have an option to 'watch' repos from within the Project for 'items' being created in the first place, in order to add them.

I suppose if the implementation of that was a sort of poll, rather than listening for created events, then it could solve the 'initial import' problem too.

**Comment by roll** [comment]
Absolutely! 

We're really interested in the board working (partially) as an Inbox so we don't miss any new issues or pull requests from our repos. Currently, we use Github Actions but maintaining it for 20+ repos are not easy and generally, it doesn't look to be a really safe solution (because of token sharing)

**Comment by ronnyroeller** [comment]
Similar issue here: We have a repo with 100s of backlog issues, which we need to import to start using Projects. I couldn't really find out how to do a mass import of those.
